UM/PluginRegistry.py

```diff
@@ -624,6 +625,15 @@ def _findPlugin(self, plugin_id: str) -> Optional[types.ModuleType]:
                 if current_version > highest_version:
                     highest_version = current_version
                     final_location = loc
+
+        # Move data (if any) to central storage
+        central_storage_file = os.path.join(final_location, plugin_id, "central_storage.json")
```

Comment:
Should use the file name from the `Trust` class?
```suggestion
        central_storage_file = os.path.join(final_location, plugin_id, Trust.getCentralStorageFilename())
```